function-web/src/main/java/com/future/function/web/configuration/RedisConfiguration.java

```diff
@@ -41,4 +47,25 @@ public JedisConnectionFactory jedisConnectionFactory(
     return redisTemplate;
   }
 
-}
+  @Bean
+  public RedisTemplate<String, Object> redisMqTemplate(JedisConnectionFactory jedisConnectionFactory) {
+    RedisTemplate<String, Object> redisTemplate = new RedisTemplate<>();
+    redisTemplate.setConnectionFactory(jedisConnectionFactory);
+    redisTemplate.setKeySerializer(new StringRedisSerializer());
+    redisTemplate.setValueSerializer(new Jackson2JsonRedisSerializer<>(Object.class));
```

Comment:
what about the expire time for this redis?

Reply:
no need expire time to be set up since it is used for storing user data (active chatroom data in this case) and not caching

Reply:
and it is stored in redis?

Reply:
yes since it is a temporary state. and also i used redis as message queue for every incoming chat

Reply:
ok, just making sure that the storage of this feature won't take too much space because of the limited space